server/src/graphql/schema.types.ts

```diff
@@ -12,82 +12,71 @@ export interface Scalars {
   Float: number
 }
 
-export interface Mutation {
-  __typename?: 'Mutation'
-  answerSurvey: Scalars['Boolean']
-  nextSurveyQuestion?: Maybe<Survey>
-}
-
-export interface MutationAnswerSurveyArgs {
-  input: SurveyInput
+export interface Query {
+  __typename?: 'Query'
+  party?: Maybe<Party>
+  songs: Array<Song>
 }
 
-export interface MutationNextSurveyQuestionArgs {
-  surveyId: Scalars['Int']
+export interface QueryPartyArgs {
+  partyName: Scalars['String']
+  partyPassword?: Maybe<Scalars['String']>
 }
 
-export interface Query {
-  __typename?: 'Query'
-  self?: Maybe<User>
-  surveys: Array<Survey>
-  survey?: Maybe<Survey>
+export interface Mutation {
+  __typename?: 'Mutation'
+  createParty?: Maybe<Party>
+  vote?: Maybe<VotedSong>
+  nextSong?: Maybe<Party>
 }
 
-export interface QuerySurveyArgs {
-  surveyId: Scalars['Int']
+export interface MutationCreatePartyArgs {
+  partyName: Scalars['String']
+  partyPassword?: Maybe<Scalars['String']>
 }
 
-export interface Subscription {
-  __typename?: 'Subscription'
-  surveyUpdates?: Maybe<Survey>
+export interface MutationVoteArgs {
+  partyId: Scalars['Int']
+  songId: Scalars['Int']
 }
 
-export interface SubscriptionSurveyUpdatesArgs {
-  surveyId: Scalars['Int']
+export interface MutationNextSongArgs {
+  partyId: Scalars['Int']
 }
 
-export interface Survey {
-  __typename?: 'Survey'
+export interface Party {
+  __typename?: 'Party'
   id: Scalars['Int']
   name: Scalars['String']
-  isStarted: Scalars['Boolean']
-  isCompleted: Scalars['Boolean']
-  currentQuestion?: Maybe<SurveyQuestion>
-  questions: Array<Maybe<SurveyQuestion>>
+  password?: Maybe<Scalars['String']>
+  latestTime: Scalars['String']
```

Comment:
Do you think it would be easier to keep this as an int (unix epoch time)? It might be easier to find the difference with the current time (for the background process), and I can't think of a reason it would need to be a string.

Reply:
I chose a string representation just by habit since gql uses 32 bit ints, so I think we could only represent up to around the year 2038. It doesn't really matter for our project, so if everyone want to change the type to an int for our API then I'm good with that, too.

Reply:
Oh ok that makes sense. I'm fine with leaving it as a string.